extensions/core/deployment/src/main/java/org/apache/camel/quarkus/core/deployment/CamelServiceFilterBuildItem.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.camel.quarkus.core.deployment;
+
+import io.quarkus.builder.item.MultiBuildItem;
+import org.apache.camel.quarkus.core.CamelServiceFilter;
+
+public final class CamelServiceFilterBuildItem extends MultiBuildItem {
```

Comment:
I'd like us all to document BuildItems properly. The class level BuildItem JavaDoc should make the extension developers understand what is the purpose of the given BuildItem and when/how they should use it. If we provide no JavaDoc, the extension developers are doomed to click through our BuildProcessors and collect the information investing a lot more effort.